pkg/container/container.go

```diff
@@ -297,6 +298,17 @@ func (c *Module) Run(ns string, data pkg.Container) (id pkg.ContainerID, err err
 	return pkg.ContainerID(container.ID()), nil
 }
 
+// Get logs of the container
+func (c *Module) Logs(ns string, containerID string) (logs string, err error) {
+	filePath := fmt.Sprintf("/var/cache/modules/contd/logs/%s/%s.log", ns, containerID)
```

Comment:
No hard coded paths please. Nor Sprintf when concatinating paths. instead this should be 
```suggestion
	filePath := filepath.Join(c.root, ns, containerID)
```

Reply:
updated